[2022-01-27 14:49:42,304] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: cgi_40.load_data_source 2022-01-27T14:49:41.423305+00:00 [queued]>
[2022-01-27 14:49:42,315] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: cgi_40.load_data_source 2022-01-27T14:49:41.423305+00:00 [queued]>
[2022-01-27 14:49:42,315] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-01-27 14:49:42,315] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-01-27 14:49:42,315] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-01-27 14:49:42,325] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): load_data_source> on 2022-01-27T14:49:41.423305+00:00
[2022-01-27 14:49:42,335] {standard_task_runner.py:52} INFO - Started process 99 to run task
[2022-01-27 14:49:42,343] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cgi_40', 'load_data_source', '2022-01-27T14:49:41.423305+00:00', '--job-id', '16', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/cgi40_dag.py', '--cfg-path', '/tmp/tmpf2hfqy0m', '--error-file', '/tmp/tmpjhlvnsu2']
[2022-01-27 14:49:42,344] {standard_task_runner.py:77} INFO - Job 16: Subtask load_data_source
[2022-01-27 14:49:42,377] {logging_mixin.py:104} INFO - Running <TaskInstance: cgi_40.load_data_source 2022-01-27T14:49:41.423305+00:00 [running]> on host bb053c5f0074
[2022-01-27 14:49:42,411] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=oulespaul@gmail.com
AIRFLOW_CTX_DAG_OWNER=ETDA
AIRFLOW_CTX_DAG_ID=cgi_40
AIRFLOW_CTX_TASK_ID=load_data_source
AIRFLOW_CTX_EXECUTION_DATE=2022-01-27T14:49:41.423305+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-01-27T14:49:41.423305+00:00
[2022-01-27 14:49:42,411] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-01-27 14:49:42,412] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'pwd']
[2022-01-27 14:49:42,421] {subprocess.py:75} INFO - Output:
[2022-01-27 14:49:42,423] {subprocess.py:79} INFO - /tmp/***tmp2saso98t
[2022-01-27 14:49:42,423] {subprocess.py:83} INFO - Command exited with return code 0
[2022-01-27 14:49:42,443] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=cgi_40, task_id=load_data_source, execution_date=20220127T144941, start_date=20220127T144942, end_date=20220127T144942
[2022-01-27 14:49:42,461] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-01-27 14:49:42,471] {local_task_job.py:151} INFO - Task exited with return code 0
